apple/testing/default_runner/ios_xctestrun_runner.template.sh

```diff
@@ -83,6 +94,9 @@ if [[ -n "$test_host_path" ]]; then
   fi
 fi
 
+test_tmp_dir_test_host_path=$(find "$test_tmp_dir" -name "*.app" -type d -maxdepth 1 -mindepth 1 -print -quit)
+test_host_name=$(basename_without_extension "$test_tmp_dir_test_host_path")
```

Comment:
Can this use `test_host_path="%(test_host_path)s"` instead? If not, mind adding a comment about why it needs to exist alongside `test_host_path`? I'd guess it handles the case where `$test_host_path` is an IPA but figured I'd ask.